samples/msgext-unfurling-ac-loop-components/csharp/assets/sample.json

```diff
@@ -0,0 +1,68 @@
+[
+  {
+    "name": "officedev-microsoft-teams-samples-msgext-unfurling-ac-loop-components-csharp",
+    "source": "officeDev",
+    "title": "Adaptive Card based loop components",
+    "shortDescription": "This sample shows messaging extension with adaptive card-based loop components functionality.",
+    "url": "https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/msgext-unfurling-ac-loop-components/csharp",
+    "longDescription": [
+      "This sample shows messaging extension with adaptive card-based loop components functionality."
+    ],
+    "creationDateTime": "2023-09-08",
```

Comment:
We can update these dates here and Readme file as well.

Reply:
Fixed